addons/sale/tests/test_sale_to_invoice_and_to_be_invoiced.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import common
+
+
+class TestSaleOrderInvoicing(common.TransactionCase):
```

Comment:
should be an AccountingTestCase, then you can remove most of the creation of master data (accounts, journals...)